typescript/cli/src/deploy/warp.ts

```diff
@@ -475,16 +489,30 @@ export async function runWarpRouteApply(params: ApplyParams): Promise<void> {
           );
           const transactions = await evmERC20WarpModule.update(config);
 
-          if (transactions.length) {
-            for (const transaction of transactions) {
-              await multiProvider.sendTransaction(chain, transaction);
-            }
-
-            logGreen(`Warp config updated on ${chain}.`);
-          } else {
-            logGreen(
+          if (transactions.length == 0)
+            return logGreen(
               `Warp config on ${chain} is the same as target. No updates needed.`,
             );
+
+          const transactionReceipts = await submitter.submit(...transactions);
+
+          if (transactionReceipts && transactionReceipts.length > 0) {
+            if (params.chainSubmissionStrategy)
+              return logGreen(
+                `✅ Warp config update successfully submitted to safe (${params.chainSubmissionStrategy}) on ${chain}:\n\n`,
```

Comment:
```suggestion
                `✅ Warp config update successfully submitted (${params.chainSubmissionStrategy}) on ${chain}:\n\n`,
```

Reply:
feels a little odd to have this branch in here
when would we be calling `submitter.submit` without a submission strategy?

Reply:
you're right. we don't need this extra branch